admiral/common_functions.sqf

```diff
@@ -93,14 +93,13 @@ adm_common_fnc_setGear = {
 adm_common_fnc_tryRemoveNVGs = {
```

Comment:
I would rename the function to something like `adm_common_fnc_assignNVG` (also removes, might not be the best).